docker-compose.yaml

```diff
@@ -1,8 +1,8 @@
 version: '3.1'
 services:
   hugo:
-      image: klakegg/hugo:0.111.0-ext-ubuntu
-      command: server -D --poll 700ms
+      image: hugomods/hugo:${HUGO_VERSION-git-0.120.4}
```

Comment:
Why did you add `-git-0.120.4`? I would have either the hardcoded number or the variable as ${HUGO_VERSION}.

Reply:
It's used to set the default value for `HUGO_VERSION` to `git-0.120.4` version, which contains the latest Hugo version with the Git extension (required to fill the "last updated" information in pages footer).

So, if users need to override Hugo version, they can:
- Export the `HUGO_VERSION` environment variable
- Use a `.env` file

See [docker-compose documentation](https://docs.docker.com/compose/environment-variables/set-environment-variables/#compose-file)

Reply:
My bad, I was more used to `${VAR:-default}` syntax which avoid taking the user variable if it's empty, and didn't know this alternative ([full list here](https://docs.docker.com/compose/environment-variables/env-file/#:~:text=VAR-,Default%20value,-%24%7BVAR%3A%2Ddefault%7D)) . I don't know which one is better in this context.